### PR TITLE
chore(api) : code formatted & quarries added

### PR DESCRIPTION
**Pull request by soumyadip007**
Subject: chore(api) : code formatted & quarries added
Assignees: @soumyadip007 

## Closes / Fixes 

1.  Core formatted after the prettier version updating
2.  Roll back to queries for the created by in deployment api

## Does this PR introduce a breaking change

NO

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api/src/server/sot/rover/rover.controller.ts [hunk 1]
```diff
@@ -18,12 +18,11 @@ export class RoverController {
...
 
-  @Get('/rover/group/:groupname')
+  @Get('/rover/group/:groupName')
   @ApiCreatedResponse({ status: 201, description: 'List for the users in Rover Group.', type: Subject, isArray: true })
   @ApiOperation({ description: 'Search the user details from Rover Group.' })
-  async getRoverGroupDetails(@Param('groupname') groupname: string, @Query('list') list: string,): Promise<Subject[]> {
-    if (list === 'group') return this.searchService.getRoverGroupList(groupname);
-    return this.searchService.getRoverGroupDetails(groupname);
+  async getRoverGroupDetails(@Param('groupName') groupName: string, @Query('list') list: string): Promise<Subject[]> {
+    if (list === 'group') return this.searchService.getRoverGroupList(groupName);
```
Comment (arkaprovob) [inline packages/api/src/server/sot/rover/rover.controller.ts:25]: 'goup' seems to be a string, and list === 'group' is confusing. Can you re-write this?

**Review by arkaprovob — CHANGES_REQUESTED** [review]

File: packages/api/src/server/sot/rover/rover.controller.ts (hunk 1 above)
Comment (soumyadip007) [reply packages/api/src/server/sot/rover/rover.controller.ts:25]: as of now there is a single Attribute, in future we can have a comma separated query in the list, so 'list' was added. 

what about 'searchBy === group' !?

Comment (arkaprovob) [reply packages/api/src/server/sot/rover/rover.controller.ts]: can you please change   to `searchByCriteria === 'group'`

Comment (soumyadip007) [reply packages/api/src/server/sot/rover/rover.controller.ts]: Hi @arkaprovob , I've made the changes, please check. this is also noted, from the next time onwards, we'll follow this practice.

**Review by arkaprovob — APPROVED** [review]

**Pull request merged**